chapter3/astroml_chapter3_Univariate_Distribution_Functions.ipynb

```diff
@@ -0,0 +1,1595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ca06a06b",
+   "metadata": {},
+   "source": [
+    "# Univariate Distribution Functions \n",
+    "## A note on the following graphs\n",
+    "\n",
+    "The code below ensures the fonts in plots are rendered with LaTex.\n",
+    "This function adjusts matplotlib settings for a uniform feel in the textbook.\n",
+    "Note that with `usetex=True`, fonts are rendered with LaTeX. This may result in an error if LaTeX is not installed on your system. In that case, you can set `usetex` to `False`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "84da7c92",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if \"setup_text_plots\" not in globals():\n",
+    "    from astroML.plotting import setup_text_plots\n",
+    "setup_text_plots(fontsize=8, usetex=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a2970784",
+   "metadata": {},
+   "source": [
+    "For each of these distributions, both `Numpy` and `Matplotlib` are needed to generate the graphs. Each graph will also require a different module from `scipy.stats` which we will import separately as you will see below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0aa41431",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf29a436",
+   "metadata": {},
+   "source": [
+    "# The uniform distribution\n",
```

Comment:
The one big title now works and is being picked up as intended, but you will need to move all these headings one layer down (## should work I suppose), so they are not being picked up for the big table of content, which currently looks like this (you can check it in the CI link at the bottom of the PR, too):

<img width="545" alt="Screenshot 2022-06-27 at 10 50 51" src="https://user-images.githubusercontent.com/6788290/176004759-4271f764-6205-40f0-8f74-5a8ae984bcd8.png">